controller.py

```diff
@@ -0,0 +1,64 @@
+# Разработал: Артем Анатольевич Авдеев
+# `controller.py` - модуль, отвечающий за управление данными и координацию между моделью и представлением.
+# Он содержит основную функцию программы, которая запускает цикл обработки действий пользователя.
+
+
+import os
+import model
+import view
+
+# Основная функция программы
+def main():
+    # Определение пути к файлу контактов
+    file_name = os.path.join(os.path.dirname(os.path.abspath(__file__)), "contacts.csv")
```

Comment:
Что-то вы тут намудрили. Получить текущую директорию можно намного проще
```suggestion
    file_name = os.path.join(os.getcwd(), "contacts.csv")
```